www/_posts/2016-02-29-docs-improvement.md

```diff
@@ -0,0 +1,31 @@
+--- 
+layout: post 
+author: 
+    name: Raghav Katyal
+    url: https://github.com/rakatyal
+title:  "Docs Improvement"
+categories: announcements 
+tags: news releases docs 
+---
+
+We are happy to announce that we have made significant content updates to our documentation.
+<!--more-->
+
+A while back we conducted a [survey](https://apachecordovabot.typeform.com/to/BCc5co) on the Cordova website asking for the experience of working with Cordova from our users. We received more than 900 responses and Documentation emerged as one of the major pain points of an average Cordova developer.
+
+Since then we have made efforts to improve the quality of our documentation. To summarize it:
+
+- Validated and corrected most of the current doc content.
+- Re-organized the content for our website and made our table of contents more concise and composed of action driven items for ease of navigation.
```

Comment:
It will be great to give a high level walk through of the different sections of the TOC: 
 - Architecture overview
- Create apps:  Focused on app developer blah blah
- Create Plugins: Focused on plugin developer blah blah
- Reference topics